test/Sentry.Unity.Tests/SentryUnityOptionsExtensionsTests.cs

```diff
@@ -88,5 +90,27 @@ public void ShouldInitializeSdk_NotCaptureInEditorAndApplicationIsEditor_Returns
 
             Assert.IsFalse(shouldInitialize);
         }
+
+        [Test]
+        public void SetupLogging_DebugTrue_SetsUnityLogger()
```

Comment:
Could have a test that checks the code path of having already a loggers set. With Debug true and false.